docs/assets/js/docs.js

```diff
@@ -1,16 +1,22 @@
-m_triggers = document.querySelectorAll("[data-trigger-modal]");
+/* Modals */
+const modalTriggers = document.querySelectorAll("[data-trigger-modal]");
 
-for(var i=0; i < m_triggers.length; i++) {
-  mt = m_triggers[i];
+for (const i = 0; i < modalTriggers.length; i++) {
+  modalTrigger = modalTriggers[i];
 
-  mt.addEventListener("click", function(e) {
+  modalTrigger.addEventListener("click", function (e) {
```

Comment:
Missing `const` on both of these assignments.

Reply:
Ignore this - GH showed me the wrong diff for L5-7.

Reply:
Fixed, by removing the additional variable by direct access